apps/web/src/styles/5-elements/_elements.page.scss

```diff
@@ -20,3 +20,7 @@ body {
 		scroll-behavior: smooth;
 	}
 }
+
+dd {
```

Comment:
I think this goes better in the generic folder in a file `_generic.reset.scss` perhaps.

Reply:
Ok... as an observation, I think we have too many folders/files for sass right now. I think the structure lends itself more to a project where we're building all the CSS, but in relying on 99% of it from GDS, it felt overly-fragmented to me...

Reply:
It is :) I was expecting way more custom stuff to begin with and I planned for that. But I think as we move fwd long term from what I have seen we will build more custom stuff and it will make sense. Refactoring later will be harder IMHO than using it like this now.